portfolio/src/main/webapp/script.js

```diff
@@ -36,4 +36,36 @@ function music(){
     //Add 2 page
     const quoteContainer = document.getElementById('greeting-container');
     quoteContainer.innerText = song;
-}
\ No newline at end of file
+}
+
+function getData(){
+    console.log('Fetching a data.'); 
+    // The fetch() function returns a Promise because the request is asynchronous.
+    const responsePromise = fetch('/data');
+    // When the request is complete, pass the response into handleResponse().
+    responsePromise.then(handleResponse);
+}
+
+/**
+ * Handles response by converting it to text and passing the result to
+ * addQuoteToDom().
```

Comment:
I think you updated the function name below, but didn't update this comment :)

Reply:
Thanks for the catch!